test/adapters/level_zero/multi_device_event_cache_tests.cpp

```diff
@@ -40,12 +54,12 @@ TEST_F(urMultiQueueMultiDeviceEventCacheTest,
     ASSERT_SUCCESS(urDevicePartition(devices[0], &properties, numSubDevices,
                                      sub_devices.data(), nullptr));
     uur::raii::Context context1 = nullptr;
-    ASSERT_SUCCESS(
-        urContextCreate(1, &sub_devices[0], nullptr, context1.ptr()));
+    ASSERT_SUCCESS(urContextCreate(sub_devices.size(), &sub_devices[0], nullptr,
+                                   context1.ptr()));
     ASSERT_NE(nullptr, context1);
     uur::raii::Context context2 = nullptr;
-    ASSERT_SUCCESS(
-        urContextCreate(1, &sub_devices[1], nullptr, context2.ptr()));
+    ASSERT_SUCCESS(urContextCreate(sub_devices.size(), &sub_devices[0], nullptr,
```

Comment:
Just noticed this, So, you meant to change this to not use the second sub device?

Reply:
line 67 would then be creating the queue on context 2 which is on sub device 0, with the queue on sub device 1

Reply:
We're still creating the second queue on the second context, and the second sub_device:
```
ASSERT_SUCCESS(urQueueCreate(context2, sub_devices[1], 0, &queue2));
```
The difference is just that both contexts now contain all subdevices. Alternative would be to just use a single context but I don't know if that matches the use-case this test was supposed to validate.

Reply:
ahh, I understand, I missed the count, then it looks good to me.